data-prepper-plugins/opensearch/src/main/resources/metrics-policy-with-ism-template.json

```
@@ -0,0 +1,22 @@
+{
+  "policy": {
+    "description": "Managing metrics for metric analytics",
+    "default_state": "current_write_index",
+    "states": [
+      {
+        "name": "current_write_index",
+        "actions": [
+          {
+            "rollover": {
+              "min_size": "50gb",
+              "min_index_age": "24h"
+            }
+          }
+        ]
+      }
+    ],
+    "ism_template": {
+      "index_patterns": ["metrics-otel-v1"]
```

```suggestion
      "index_patterns": ["metrics-otel-v1-*"]
```

Should we use a wildcard here?